maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/ScreenDiffHandler.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import XCTest
+import CryptoKit
+import FlyingFox
+import os
+
+class ScreenDiffHandler : RouteHandler {
+    private let logger = Logger(subsystem: Bundle.main.bundleIdentifier!, category: "ScreenDiffHandler")
+    
+    func handle(request: FlyingFox.HTTPRequest) async throws -> FlyingFox.HTTPResponse {
+        let screenshot1 = XCUIScreen.main.screenshot()
```

Comment:
Can this method fail? If yes lets have log around it. Also can you test if this works well with screens which were failing due to being long (feed screens)

Reply:
this one seems to be non-failable, result is not optional and the func doesn't throw